conditional-signing/browser/src/utils.ts

```diff
@@ -0,0 +1,8 @@
+export const getEnv = (name: string) => {
```

Comment:
The point of this function is throw an error if required ENVs are passed. If the ENV isn't required, then you can just use `import.meta.env[name]`. As is this function is not different than just using `import.meta.env[name]`